data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/server/RequestExceptionHandler.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.peerforwarder.server;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+
+import java.util.Objects;
+
+public class RequestExceptionHandler {
+
+    public HttpResponse handleException(final Exception e, final String message) {
+        Objects.requireNonNull(message);
+        if (e instanceof JsonProcessingException) {
```

Comment:
I'm not sure about having this externalized from the server. I don't think a `JsonProcessingException` is necessarily a Bad Request. When that JSON comes from the client, it is. But, as the server evolves, the server may do its own JSON work that would be indicative of an Internal Server Error.